go/vt/sqlparser/sql.y

```diff
@@ -6950,26 +6958,35 @@ set_list:
   }
 
 set_expression:
-  reserved_sql_id '=' ON
+  set_variable '=' ON
```

Comment:
set_exprsesion, apparently also used to take in non-reserved and reserved keywords. It does not now. Is this change intentional?

Reply:
no, and probably wrong.